### PR TITLE
Prepostconditions

### PR DESCRIPTION
**Pull request by MichielvdBerg**
For now, only the actions that we use have pre and post conditions. The others can be added when we use them.

"As an agent
When in a simulation
I am able to build stuff if I think this 
is benificial for me.
To optimize a certain indicator or to 
reach a construction goal."

### CODE REVIEW

File: TU-Agent/tygron.act2g [hunk 1]
```diff
@@ -4,8 +4,8 @@ use agentKnowledge as knowledge.
 % Plan a new building
 % params = FunctionId, Amount of Floors, MultiPolygon describing the build contour
 define	building_plan_construction(FunctionId, Floors, MultiPolygon) with
-   pre{ true }
-   post{ true }
+   pre{ demolishedPolygon(MultiPolygon) }
+   post{ not(demolishedPolygon(MultiPolygon)) }
```
Comment (tvleest) [inline TU-Agent/tygron.act2g:8]: didn't we rename this?

File: TU-Agent/tygron.act2g [hunk 2]
```diff
@@ -26,19 +26,14 @@ define map_sell_land(StakeholderId, MultiPolygon, Price) with
...
-  post{ true }
-  
-% Define the new message action
-% Send a message between Stakeholders
-% params = Receiving Stakeholder ID, Subject, Content, Motivation needed
-define message_new(StakeholderId, Subject, Content, IsMotivationNeeded) with
-	pre{ true }
-	post { true}
+  pre{ oldbuildings([building(N,A,B,C,D,E,F,G)|Tail]) }
+  post{ oldbuildings(Tail), demolishedPolygon(G)}
```
Comment (tvleest) [inline TU-Agent/tygron.act2g:30]: also renamed

**Comment by tvleest** [comment]
If you rename the variable (probably best to merge in the changes from master) I approve.

**Comment by tvleest** [comment]
Just realized that master does not have this rename yet. I think we can merge and it'll be solved eventually. approved.

**Comment by spiderbiggen** [comment]
approved

**Pull request merged**